settings/src/components/AdminTwoFactor.vue

```diff
@@ -107,13 +117,27 @@
 			},
 			excludedGroups: {
 				get: function () {
-					return this.$store.state.excludedGroups
+					return this.$store.state.security.excludedGroups
 				},
 				set: function (val) {
 					this.dirty = true
 					this.$store.commit('setExcludedGroups', val)
 				}
 			},
+			noProviderGlobally: {
```

Comment:
You can directly use the function if you don't have a setter here.

```suggestion
			noProviderGlobally() {
				var providers = this.$store.getters.getAllApps.filter( function(app) {
					return ('two-factor-providers' in app && 'provider' in app['two-factor-providers'] && app['active'] === true);
				});
				return (providers.length === 0);
			}
```